multiview/cluster/mv_k_means.py

```diff
@@ -0,0 +1,262 @@
+import numpy as np
+from .base_cluster import BaseCluster
+from ..utils.cluster_utils import check_Xs
+
+
+class MultiviewKMeans(BaseCluster):
+
+    '''
+    An implementation of Multi-View K-Means using the co-EM algorithm.
+    This algorithm currently handles two views of data.
+
+    Paramters
+    ---------
+    k : int
+        The number of clusters
+
+    random_state : int (default=None)
+        Determines random number generation for initializing centroids.
+        Can seed the random number generator with an int.
+
+    max_iter : int (default=None)
+        The maximum number of iterations to run the EM algorithm.
+
+    Attributes
+    ----------
+
+    _k : int
+        The number of clusters
+
+    _random_state : int
+        The seed for the random number generator used during centroid
+        initialization.
+
+    _centroids : list of array_likes
+        - _centroids shape: (2,)
+        - _centroids[0] shape: (n_clusters, n_features_i)
+        The cluster centroids for each of the two views. _centroids[0]
+        corresponds to the centroids of view 1 and _cnetroids[1] corresponds
+        to the centroids of view 2.
+
+
```

Comment:
Maybe not necessary, but could add a reference to a co-em paper if there's a seminal one